modules/social_features/social_group/src/Form/SocialGroupSettings.php

```diff
@@ -138,4 +234,29 @@ protected function hasPermission($name) {
     return !empty($this->config('social_group.settings')->get($name));
   }
 
+  /**
+   * Returns node types list used as a group content.
+   *
+   * @return array
+   *   An array with options.
+   */
+  private function getCrossPostingEntityTypesOptions() {
+    // The list of node types allowed for cross-posting in groups.
+    // @todo: maybe is better to create a list of entity bundles keyed by entity type.
+    $entity_types = ['topic', 'event'];
+    // Add possibility to add entity types from other modules.
+    $this->moduleHandler->alter('cross_posting_entity_list', $entity_types);
```

Comment:
@nechai We usually start to alter hook names with module name.
So maybe we can name it `social_group_cross_posting`
Also, an example needs to be added to the `social_group.api.php` file.

Reply:
@ribel changed hook name according to your suggestions